project/Dependencies.scala

```diff
@@ -42,6 +44,7 @@ object Dependencies {
 
   val googleDataproc: ModuleID =    "com.google.apis"     % "google-api-services-dataproc" % s"v1-rev53-$googleV" excludeAll(excludeGuavaJDK5)
   val googleRpc: ModuleID = "io.grpc" % "grpc-core" % "1.5.0"
+  val googleOAuth2: ModuleID = "com.google.auth" % "google-auth-library-oauth2-http" % "0.9.0"
```

Comment:
Not for this PR, but a point of housekeeping -- this means we use two different libraries to construct credentials. At some point we should switch our other credential-building from the old one to this one entirely.